app/controllers/user_controller.rb

```diff
@@ -1,7 +0,0 @@
-class UserController < ApplicationController
-  def new
-  end
-
-  def create
-  end
-end
```

Comment:
warum hast du das gelöscht?

Reply:
In Absprache mit Toni... Sämtliche User routes sind in devise abgebildet. Unser User_controller daher redundant